tests/executors/test_celery_executor.py

```diff
@@ -288,6 +291,9 @@ def test_command_validation(self, command, expected_exception):
         with mock.patch(
             'airflow.executors.celery_executor._execute_in_subprocess'
         ) as mock_subproc, mock.patch('airflow.executors.celery_executor._execute_in_fork') as mock_fork:
+            test_app = MagicMock()
+            test_app.current_task.request.id.side_effect = "abcdef-124215-abcdef"
+            mock.patch('airflow.executors.celery_executor.app', test_app)
```

Comment:
I think we can approach this like this:
```python

with mock.patch(
            'airflow.executors.celery_executor._execute_in_subprocess'
        ) as mock_subproc, mock.patch('airflow.executors.celery_executor._execute_in_fork') as mock_fork,
        mock.patch('airflow.executors.celery_executor.app') as test_app:
            test_app.current_task.request.id.side_effect = "abcdef-124215-abcdef"
```
WDYT?

Reply:
I don't really like having two mock.patches on the same line. I choose not to use three because it becomes more unreadable the more you add. I think it's better the way it is already.

Reply:
I don't have strong opinion here too